tests/baselines/reference/constEnumPropertyAccess1.js

```diff
@@ -38,7 +38,7 @@ var o = {
     1: true
 };
 var a = 1 /* A */;
-var a1 = 1 /* "A" */;
+var a1 = 1 /* A */;

```

Comment:
why changed?

I think it caused by following to:
https://github.com/microsoft/TypeScript/blob/88f80c75e1a4ab6aaec605aa4ec6281b87871ff0/tests/baselines/reference/constEnumPropertyAccess1.js#L22